src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.impl.StorageDaoImpl;
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.handlers.TransactionHandler;
+import core.basesyntax.handlers.impl.RemoveTransactionHandler;
+import core.basesyntax.handlers.impl.SaveTransactionHandler;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.parser.TextLineParser;
+import core.basesyntax.parser.impl.CsvTextLineParser;
+import core.basesyntax.service.impl.ReaderServiceImpl;
+import core.basesyntax.service.impl.ReportServiceImpl;
+import core.basesyntax.service.impl.WriterServiceImpl;
+import core.basesyntax.strategy.TransactionStrategy;
+import core.basesyntax.strategy.impl.TransactionStrategyImpl;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String FRUITS_CSV_FILEPATH = "src/main/resources/fruits.csv";
+    private static final String REPORT_CSV_FILEPATH = "src/main/resources/report.csv";
+    private static final int HEADER_INDEX = 0;
+
+    public static void main(String[] args) {
+        List<String> fileContents = new ReaderServiceImpl().readFromFile(FRUITS_CSV_FILEPATH);
+        TextLineParser lineParser = new CsvTextLineParser();
+        FruitStorage storage = new FruitStorage();
+        Map<FruitTransaction.Operation, TransactionHandler> strategyMap = initStrategyMap(storage);
+        TransactionStrategy strategy = new TransactionStrategyImpl(strategyMap);
+
+        fileContents.remove(HEADER_INDEX);
+        for (String line : fileContents) {
+            FruitTransaction transaction = lineParser.extractTransaction(line);
+            if (transaction != null) {
+                TransactionHandler transactionHandler = strategy.get(transaction.getOperation());
+                transactionHandler.process(transaction);
+            }
+        }
```

Comment:
Let's do not place any low-level logic inside the main method. In the main method you have to just call methods of your services